src/Microsoft.Framework.Logging.TraceSource/TraceSourceLoggerProvider.cs

```diff
@@ -11,13 +11,15 @@ namespace Microsoft.Framework.Logging
     /// <summary>
     /// Provides an ILoggerFactory based on System.Diagnostics.TraceSource.
     /// </summary>
-    public class TraceSourceLoggerProvider : ILoggerProvider
+    public class TraceSourceLoggerProvider : ILoggerProvider, IDisposable
```

Comment:
@davidfowl Any update on this? Should @ycrumeyrolle follow @lodejard 's suggestion?